packages/validator-bonds-cli-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@marinade.finance/validator-bonds-cli-core",
-  "version": "2.1.0",
+  "version": "2.1.2",
```

Comment:
(non-blocking)
Just curious why it's bumped to `2.1.2` and not `2.1.1`? 🤔

Reply:
The trouble with 2.1.1 is of wrong published version in CLI. Thus it needed to be re-published :shrug: